### PR TITLE
Add: embed static HTML (spa) example

### PR DESCRIPTION
**Pull request by aprakasa**
A sample application to [embed](https://golang.org/pkg/embed/) [SvelteKit](https://kit.svelte.dev/) app with [fiber filesystem middleware](https://github.com/gofiber/fiber/tree/master/middleware/) to a single binary file.

Thanks, and I hope this is useful!

### CODE REVIEW

**Comment by ReneWerner87** [comment]
Thx, please check the bot comments

File: sveltekit-embed/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Stage 1: Build the static HTML
+FROM mhart/alpine-node:16.4.2 as frontend-builder
```
Comment (ReneWerner87) [inline sveltekit-embed/Dockerfile:2]: why not the official alpine node ?

Comment (aprakasa) [reply sveltekit-embed/Dockerfile:2]: Will change it with the official one

**Comment by aprakasa** [comment]
Hi,
I think that not possible because the js file is a development file.
Is typescript also has a restriction?

**Comment by ReneWerner87** [comment]
> triction?

its okay, we ignore the comments from the bot

please add a section in the readme in the root of the project

**Comment by aprakasa** [comment]
There is a readme file at the root of the project which include "how to run the project" section.
The other readme file is inside the frontend folder which is blank.
May I know which section is missing?

**Pull request merged**